src/components/newcases.js

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useEffect} from 'react';
+
+function RateOfNewCases(props) {
+  const [firstDayCases, setFirstDayCases] = useState(0);
+  const [firstDayDate, setFirstDayDate] = useState(0);
+  const [secondDayCases, setSecondDayCases] = useState(0);
+  const [secondDayDate, setSecondDayDate] = useState(0);
+  const [thirdDayCases, setThirdDayCases] = useState(0);
+  const [thirdDayDate, setThirdDayDate] = useState(0);
+  const [fourthDayCases, setFourthDayCases] = useState(0);
+  const [fourthDayDate, setFourthDayDate] = useState(0);
+  const [fifthDayCases, setFifthDayCases] = useState(0);
```

Comment:
Why not maintain them in an array of object?  This is just very verbose.
```
[  { cases: 10, date: '30-03-2020' } ]
``

Reply:
changed as per the comment.